pandas/core/common.py

```diff
@@ -3095,3 +3095,32 @@ def pandas_dtype(dtype):
             pass
 
     return np.dtype(dtype)
+
+
+def validate_args(args, min_length=0, max_length=None):
+    """
```

Comment:
let's move ``core/generic.py/_validate_kwargs`` here as well (and put a comment in this seconds of 2 functions to indcate the purpose)